components/admin_console/team_channel_settings/team/details/team_profile.jsx

```diff
@@ -56,6 +69,35 @@ export function TeamProfile({team}) {
 
                         </div>
                     </div>
+                    {/* We are using same classes used for Archive Channel button so that we can use the same styling as Arvhive channel button */}
```

Comment:
I personally think the comments regarding class names are not needed here. We usually want to include comments when there is logic that needs to be explained. I think we are fine without the comment here and the one below.

Though if we do include the comments, they should use the same comment style:
```suggestion
                    // We are using same classes used for Archive Channel button so that we can use the same styling as Archive channel button
```